ids_validator/checks/generic/athena.py

```diff
@@ -195,3 +192,39 @@ def get_athena_partitions_name(cls, athena_schema) -> list:
             for partition in partitions
         ]
         return names
+
+    @classmethod
+    def normalize_path_name(cls, path_name):
+        """
+        Weird-partition!@name -> weird_partition_name
+        @fileId -> fileid
+        project.name -> project_name
+        """
+        normalized_path = re.sub('[^A-Za-z0-9]+', '_', path_name)
+        normalized_path = re.sub('[_]+', '_', normalized_path)
+        if normalized_path.startswith("_"):
+            normalized_path = normalized_path[1:]
+
+        if normalized_path.endswith("_"):
+            normalized_path = normalized_path[:-1]
```

Comment:
I think these four lines could be replaced with just a `strip('_')`:
```suggestion
        normalized_path = normalized_path.strip("_")
```

Reply:
Thanks! Fixed it.